### ISSUE TITLE
Rollback to the legacy values not working.

### ISSUE THREAD

**Issue by tomplant** [ihsoft/KIS#332]
Hi, Everything's been great, and for a short time I wanted to rollback the new behavior to the legacy values while as I modify existing items to work with the new values. (Sending more containers to bases and such).

I went and located the `kis_legacy_containers.txt` file in the `KIS` installation and renamed it to `kis_legacy_containers.cfg`. However upon loading the game after this, the values on existing items, and in the VAB reflect the new limits.

Is there something I've missed or need to do additionally? 
Thanks!

**Comment by ihsoft** [comment]
Are you sure you've did it right? I've tested this file before adding it into the release, and I can hardly imagine how it cannot work. Please, share your `KIS` folder snapshot as well as the [KSP log](https://github.com/ihsoft/KIS/wiki/Troubleshooting#how-to-get-the-games-log-file). Note, that I need a very specific log (see the link).

**Comment by tomplant** [comment]
Thanks for getting back on this. I checked my work a fourth time, and I guess either having the .cfg and .txt file in the same folder is an issue, or I goofed the first couple of times I did it. After doing it again, the values reverted- which is great because it's giving me a moment to get more containers to my 3 bases in Career mode. 
Also, was the volume for Kerbals themselves change? I seem to think it was, but the legacy containers doesn't switch that. That could be just in my mind.

Regardless, thanks for the great mod!
